oak-search/src/test/java/org/apache/jackrabbit/oak/plugins/index/IndexSuggestionCommonTest.java

```diff
@@ -444,6 +444,35 @@ public void testDuplicateSuggestionProperties() throws Exception {
         });
     }
 
+    @Test
+    public void testDuplicateSuggestions() throws Exception {
+        String nodeType = JcrConstants.NT_UNSTRUCTURED;
+        String suggestProp1 = "des";
+
+        createSuggestIndex("index-suggest", nodeType, suggestProp1);
+
+        Node indexedNode = root.addNode("indexedNode", nodeType);
+        indexedNode.setProperty(suggestProp1, "Hello World");
+        indexedNode = root.addNode("indexedNode2", nodeType);
+        indexedNode.setProperty(suggestProp1, "Hello World");
+        indexedNode = root.addNode("indexedNode3", nodeType);
+        indexedNode.setProperty(suggestProp1, "Hello Oak");
+
+        session.save();
+
+        String suggQuery = createSuggestQuery(nodeType, "Hello");
+        QueryManager queryManager = session.getWorkspace().getQueryManager();
+        assertEventually(() -> {
+            try {
+                List<String> resultSet = getAllResults(queryManager, suggQuery);
+                assertEquals("There should not be any duplicate suggestions, Expected - 2, Actual - " + resultSet.size(),
```

Comment:
`assertEquals` already prints `expected:<x> but was:<y>`

```suggestion
                assertEquals("There should not be any duplicate suggestions",
```

Reply:
done